### PR TITLE
Move UserView class to Infrastructure layer

### LINKED ISSUE #22: Wrong namespace for App\Domain\User\Query ?!
Hi @jorge07 ,

First of all many thanks for taking the time to demonstrate a DDD-es-cqrs example with sf4. I use it to start up a new project, and doing ddd for some time but not sf4, it does help a lot.

However, I'm a bit sceptical about the location of the classes in App\Domain\User\Query ... Imho they should not be in the Domain namespace. Generally what we will get for the "Query" part are projections and only data necessary for views, right?!

I guess the Read repository interface is acceptable to keep in the Domain layer, but the UserView class seems not right to me ...

What's your thought on this?

Thanks

### PR DESCRIPTION
**Pull request by Lutacon**
Fixes #22

I did the refactor of moving UserView to infrastructure layer talked on #22.

I have separated orm mappings into domain & infrastructure. Also, I think YAML mappings are going to be deprecated in new versions of Doctrine in favor of XML. We could also migrate the YAML files to XML.

### CODE REVIEW

File: config/packages/doctrine.yaml [hunk 1]
```diff
@@ -27,6 +27,12 @@ doctrine:
             Domain:
                 is_bundle: false
                 type: yml
-                dir: '%kernel.project_dir%/config/mapping/orm/query'
+                dir: '%kernel.project_dir%/config/mapping/orm/domain'
```
Comment (jorge07) [inline config/packages/doctrine.yaml]: we can remove domain from orm config as is not one of domain responsibilities

**Comment by jorge07** [comment]
I added travis integration so, please rebase to see how ci goes ;)

File: config/packages/doctrine.yaml (hunk 1 above)
Comment (Lutacon) [reply config/packages/doctrine.yaml]: What do you propose? deleting the folder domain and moving the mapping of the VO to infrastructure?

Comment (jorge07) [reply config/packages/doctrine.yaml]: Yes, exactly that.

**Comment by jorge07** [comment]
Well, not a rebase you did a merge.

**Comment by Lutacon** [comment]
LOL. Yes, I did a merge instead of a rebase 😢 . My bad. I will fix it

File: config/packages/doctrine.yaml (hunk 1 above)
Comment (Lutacon) [reply config/packages/doctrine.yaml]: If I do that. I will need to rename the mappings files to include Domain/Infrastructure when necessary.

I'm going to do a commit with that change and we'll review it.

File: config/packages/doctrine.yaml [hunk 2]
```diff
@@ -24,9 +24,9 @@ doctrine:
...
         auto_mapping: true
         mappings:
-            Domain:
+            Infrastructure:
                 is_bundle: false
                 type: yml
-                dir: '%kernel.project_dir%/config/mapping/orm/query'
-                prefix: 'App\Domain'
-                alias: Domain
+                dir: '%kernel.project_dir%/config/mapping/orm/infrastructure'
```
Comment (jorge07) [inline config/packages/doctrine.yaml]: what about `projections` instead `infrastructure `?

**Comment by jorge07** [comment]
LGTM just one concert about the folder name

File: config/packages/doctrine.yaml (hunk 2 above)
Comment (Lutacon) [reply config/packages/doctrine.yaml]: That is a much better name. I'm going to change it and squash commits

**Comment by Lutacon** [comment]
I fucked up with the rebase. I'm gonna open a new PR later.

**Pull request closed without merging**